homeassistant/components/terncy/translations/zh-Hant.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Don't add additional languages in the initial PR. Additional translations are managed using our translation platform.

Please remove this file.